tests/data/test1117

```diff
@@ -32,7 +32,7 @@ partial body
 </data1>
 
 <servercmd>
-writedelay: 1000
+writedelay: 100
```

Comment:
100 msec might not be enough to guarantee a context switch to the client in that time on a really loaded system. I believe this delay will only happen a single time so it needs to be long enough to make a difference.

Reply:
I'll revert this again. The saving does not seem to justify the loss of reliability.